src/harness/fourslash.ts

```diff
@@ -2508,6 +2508,20 @@ namespace FourSlash {
             }
         }
 
+        public verifySpanOfEnclosingComment(negative: boolean, onlyMultiLine: boolean) {

```

Comment:
[Methods in `VerifyNegatable`](https://github.com/Microsoft/TypeScript/blob/eb8f7bddc2a1438d59846b9b5510f98310f6ab84/src/harness/fourslash.ts#L3615-L3665) often call into the [`TestState`](https://github.com/Microsoft/TypeScript/blob/eb8f7bddc2a1438d59846b9b5510f98310f6ab84/src/harness/fourslash.ts#L182) by passing in `negative`. I'm merely following that idiom.